google-cloud-examples/src/test/java/com/google/cloud/examples/logging/snippets/ITLoggingSnippets.java

```diff
@@ -45,11 +46,11 @@
   private static final String DATASET = "dataset";
   private static final Set<String> DESCRIPTOR_TYPES = ImmutableSet.of("gce_instance", "gae_app",
```

Comment:
There is probably nothing that can be done that is correct, but it would seem like this would be ideal at the start of the snippet rather than in the tests.